react/src/components/organisms/TabContainer/tab.js

```diff
@@ -3,109 +3,113 @@ import PropTypes from 'prop-types';
+            && e.currentTarget.closest('div.ma__tab-container').parentElement.closest('div.ma__tab-container').getElementsByClassName('ma__tab-title--active')[0].previousElementSibling) {
+            const prevTab = e.currentTarget
+              .closest('div.ma__tab-container')
+              .parentElement
+              .closest('div.ma__tab-container')
+              .getElementsByClassName('ma__tab-title--active')[0]
+              .previousElementSibling
+              .getElementsByTagName('button')[0];
+            prevTab.removeAttribute('tabindex');
+            prevTab.focus();
           }
-        };
-        const { setActiveTab, activeTab } = context;
-        const tabClasses = classNames({
-          'ma__tab-title': true,
-          'ma__tab-title--active': props.default || active
-        });
-        const buttonProps = {
-          onClick: (e) => {
-            e.target.scrollIntoView({ behavior: 'smooth', block: 'nearest', inline: 'center' });
-            if (activeTab !== tabIdent) {
-              setActiveTab(tabIdent, props.children);
-            }
-            if (is.fn(props.handleClick)) {
-              props.handleClick(e, tabIdent, props.children);
-            }
-          },
-          onKeyDown: handleKeyDown,
-          id: tabIdent,
-          'aria-selected': active,
-          'aria-controls': context.tabContainerBodyId,
-          role: 'tab',
-          ref,
-          tabIndex: active ? 0 : -1
-        };
-        return(
-          <li role="presentation" className={tabClasses}>
-            <button {...buttonProps}>{props.title}</button>
-          </li>
-        );
+        }
       }
     }
-  </TabContext.Consumer>
-));
+  };
+  render() {
+    const { tabIdent, active } = this.props;
+    const { setActiveTab, activeTab } = this.context;
+    const tabClasses = classNames({
+      'ma__tab-title': true,
+      'ma__tab-title--active': active
+    });
+    const buttonProps = {
+      onClick: (e) => {
+        e.target.scrollIntoView({ behavior: 'smooth', block: 'nearest', inline: 'center' });
+        e.persist();
+        if (activeTab !== tabIdent) {
+          setActiveTab(tabIdent);
+        }
+        if (is.fn(this.props.handleClick)) {
+          this.props.handleClick(e, tabIdent, this.props.children);
+        }
+      },
+      onKeyDown: this.handleKeyDown,
+      id: tabIdent,
+      'aria-selected': active,
+      'aria-controls': this.context.tabContainerBodyId,
+      role: 'tab',
+      ref: this.props.tabRef,
+      tabIndex: active ? 0 : -1
+    };
+    return(
+      <React.Fragment>
+        <li role="presentation" className={tabClasses}>
+          <button {...buttonProps}>{this.props.title}</button>
+        </li>
+        {global.window && (
+          <TabBody active={activeTab === tabIdent} tabContainerBodyId={this.context.tabContainerBodyId}>
+            {this.props.children}
```

Comment:
The bug is fixed! I'm curious why you think this should use portal, as the react tree structure is the same as the DOM tree. In the doc portal is used specifically if we want the structure to be different